entity-framework/core/what-is-new/ef-core-9.0/breaking-changes.md

```diff
@@ -80,6 +81,33 @@ Not having matching number of arguments and nullability propagation arguments ca
 
 Make sure the `argumentsPropagateNullability` has same number of elements as the `arguments`. When in doubt use `false` for nullability argument.
 
+<a name="nullable-tostring"></a>
+
+### `ToString()` method now returns empty string for arguments with `null` values
```

Comment:
Technically we're not talking about arguments to `ToString()` here, but rather the instance (so change below as well?)

```suggestion
### `ToString()` method now returns empty string for `null` instances
```